core/common/src/rpc/web/WebAppRpcRequest.ts

```diff
@@ -189,7 +190,7 @@ export class WebAppRpcRequest extends RpcRequest {
     const requestClass = this.supplyRequest();
     const fetchFunction = this.supplyFetch();
 
-    const path = new URL(this.path, typeof (location) !== "undefined" ? location.origin : undefined);
+    const path = new URL(this.path, location?.origin ?? "undefined");
```

Comment:
would prefer window?.location?.origin

Reply:
I wasn't sure if `window.location` is always the same as `location`... is it?

Reply:
yes it should be, and its best to be safer
https://stackoverflow.com/questions/4709037/window-location-versus-just-location